jiant/tasks/lm.py

```diff
@@ -172,3 +179,173 @@ def __init__(self, path, *args, **kw):
             "val": os.path.join(path, "valid.sentences.txt"),
             "test": os.path.join(path, "test.sentences.txt"),
         }
+
+
+@register_task("wikipedia_corpus_mlm", rel_path="wikipedia_corpus_small/")
+class MaskedLanguageModelingTask(Task):
+    """
+    Masked language modeling task on Wikipedia dataset
+    Attributes:
+        max_seq_len: (int) maximum sequence length
+        min_seq_len: (int) minimum sequence length
+        files_by_split: (dict) files for three data split (train, val, test)
+    We are currently using an unpreprocessed version of the Wikipedia corpus
+    that consists of 5% of the data. You can generate the data using code from
+    https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/LanguageModeling/BERT/data.
```

Comment:
@pyeres here's the instructions for data generation (point 2 you raised).

Reply:
Thanks — it looks like the link points to a directory with a number of scripts. Should these instructions be made more specific to help the user reproduce this task?